src/components/Lesson/Progress.scss

```diff
@@ -0,0 +1,14 @@
+.marginTop {
+  margin-top: 20px;
+
```

Comment:
Legg til `position: relative` for å få teksten sentrert (altså for å få `.label` sin `position: absolute` til å virke)